src/Core/Checkout/Document/Renderer/CreditNoteRenderer.php

```diff
@@ -189,16 +202,21 @@ public function getDecorated(): AbstractDocumentRenderer
         throw new DecorationPatternException(self::class);
     }
 
-    private function getOrder(DocumentGenerateOperation $operation, Context $context, DocumentRendererConfig $rendererConfig): OrderEntity
-    {
-        ['language_id' => $languageId] = $this->getOrdersLanguageId([$operation->getOrderId()], Defaults::LIVE_VERSION, $this->connection)[0];
+    private function getOrder(
+        DocumentGenerateOperation $operation,
+        string $versionId,
+        Context $context,
+        DocumentRendererConfig $rendererConfig
+    ): OrderEntity {
+        ['language_id' => $languageId] = $this->getOrdersLanguageId([$operation->getOrderId()], $versionId, $this->connection)[0];
```

Comment:
Nitpick as it was also like this before: I was a little confused at first reading this line but turns out it's just the array de-structuring in PHP. But as you only need one field anyways I would still prefer the old way:
```suggestion
        $languageId = $this->getOrdersLanguageId([$operation->getOrderId()], $versionId, $this->connection)[0]['language_id'];
```

Reply:
yes, looks much better 👍🏻